src/main/java/org/generationcp/middleware/dao/dms/ProjectPropertyDao.java

```diff
@@ -279,6 +273,15 @@ public void deleteProjectVariables(final Integer projectId, final List<Integer>
 		query.executeUpdate();
 	}
 
+	public void deleteProjectVariablesByVariableTypes(final Integer projectId, final List<Integer> variableTypeIds) {
```

Comment:
I think we should start standardizing our concepts and start using dataset instead of project whenever is possible. It makes sense to rename this by deleteDatasetVariablesByVariableTypes.